api/metrics_service.proto

```diff
@@ -39,7 +39,6 @@ message StreamMetricsMessage {
 
 // Configuration structure.
```

Comment:
Can you add a leading comment explaining how this is supposed to be used? I.e. that it's a stats sink opaque config and provide an internal `:ref:` link back to the stats sink field where this is defined (https://github.com/envoyproxy/data-plane-api/blob/master/api/stats.proto#L28).

Also, should this be named `MetricsServiceSink` to be consistent with the other sink config in https://github.com/envoyproxy/data-plane-api/blob/master/api/stats.proto#L127?